ee/backup/tests/minio/backup_test.go

```diff
@@ -0,0 +1,290 @@
+	require.NoError(t, err)
+
+	// After the move, we need to pause a bit to give zero a chance to quorum.
+	t.Log("Pausing to let zero move tablet...")
+	moveOk := false
+	for retry := 5; retry > 0; retry-- {
+		time.Sleep(3 * time.Second)
+		state, err := z.GetState()
+		require.NoError(t, err)
+		if _, ok := state.Groups["1"].Tablets["movie"]; ok {
+			moveOk = true
+			break
+		}
+	}
+	require.True(t, moveOk)
+
+	// Setup test directories.
+	dirSetup()
+
+	// Send backup request.
+	dirs := runBackup(t, 3, 1)
+	restored := runRestore(t, dirs[0], math.MaxUint64)
+
+	checks := []struct {
+		blank, expected string
+	}{
+		{blank: "x1", expected: "BIRDS MAN OR (THE UNEXPECTED VIRTUE OF IGNORANCE)"},
+		{blank: "x2", expected: "Spotlight"},
+		{blank: "x3", expected: "Moonlight"},
+		{blank: "x4", expected: "THE SHAPE OF WATERLOO"},
+		{blank: "x5", expected: "BLACK PUNTER"},
+	}
+	for _, check := range checks {
+		require.EqualValues(t, check.expected, restored[original.Uids[check.blank]])
+	}
+
+	// Add more data for the incremental backup.
+	incr1, err := dg.NewTxn().Mutate(ctx, &api.Mutation{
+		CommitNow: true,
+		SetNquads: []byte(fmt.Sprintf(`
+			<%s> <movie> "Birdman or (The Unexpected Virtue of Ignorance)" .
+			<%s> <movie> "The Shape of Waterloo" .
+		`, original.Uids["x1"], original.Uids["x4"])),
+	})
+	t.Logf("%+v", incr1)
+	require.NoError(t, err)
+	time.Sleep(5 * time.Second)
+
+	// Perform first incremental backup.
+	dirs = runBackup(t, 6, 2)
+	restored = runRestore(t, dirs[1], incr1.Context.CommitTs)
+
+	checks = []struct {
+		blank, expected string
+	}{
+		{blank: "x1", expected: "Birdman or (The Unexpected Virtue of Ignorance)"},
+		{blank: "x4", expected: "The Shape of Waterloo"},
+	}
+	for _, check := range checks {
+		require.EqualValues(t, check.expected, restored[original.Uids[check.blank]])
+	}
+
+	// Add more data for a second incremental backup.
+	incr2, err := dg.NewTxn().Mutate(ctx, &api.Mutation{
+		CommitNow: true,
+		SetNquads: []byte(fmt.Sprintf(`
+				<%s> <movie> "The Shape of Water" .
+				<%s> <movie> "The Black Panther" .
+			`, original.Uids["x4"], original.Uids["x5"])),
+	})
+	require.NoError(t, err)
+	time.Sleep(5 * time.Second)
+
+	// Perform second incremental backup.
+	dirs = runBackup(t, 9, 3)
+	restored = runRestore(t, dirs[2], incr2.Context.CommitTs)
+
+	checks = []struct {
+		blank, expected string
+	}{
+		{blank: "x4", expected: "The Shape of Water"},
+		{blank: "x5", expected: "The Black Panther"},
+	}
+
+	for _, check := range checks {
+		require.EqualValues(t, check.expected, restored[original.Uids[check.blank]])
+	}
+
+	// Add more data for a second full backup.
+	incr3, err := dg.NewTxn().Mutate(ctx, &api.Mutation{
+		CommitNow: true,
+		SetNquads: []byte(fmt.Sprintf(`
+				<%s> <movie> "El laberinto del fauno" .
+				<%s> <movie> "Black Panther 2" .
+			`, original.Uids["x4"], original.Uids["x5"])),
+	})
+	require.NoError(t, err)
+	time.Sleep(5 * time.Second)
```

Comment:
Do we plan to add an endpoint to query whether backup is complete? This doesn't seem sustainable.